README.md

```diff
@@ -240,10 +81,6 @@ Issue Tracker: github.com/docker/distribution/issues
 Google Groups: https://groups.google.com/a/dockerproject.org/forum/#!forum/distribution
```

Comment:
This doesn't format correctly, which I think it never did (my fault). Should be a dl or list with formatted entries.